gettsim/parameters/grunds_ia.yaml

```diff
@@ -121,3 +121,11 @@ prv_rente_anr_frei:
       2:
         upper_threshold: inf
         rate: 0.3
+
+mehrbedarf_g:
+  description:
```

Comment:
@LauraGergeleit, can you improve the documentation of this parameter in a similar fashion as you did for the other retirement parameters?